awriter/signer.go

```diff
@@ -0,0 +1,140 @@
+// Copyright 2019 Northern.tech AS
```

Comment:
This file(s) should not be part of the PR. Please rebase from mendersoftware/master

Reply:
tried this, unfortunately rebasing doesn't help - I guess I'll just close this one and reopen from a new branch